sdk/textanalytics/test-resources.json

```diff
@@ -43,10 +43,15 @@
         "endpointPrefix": {
             "defaultValue": "textanalytics",
             "type": "string"
+        },
+        "endpointSuffix": {
+            "defaultValue": ".cognitiveservices.azure.com",
+            "type": "string"
         }
     },
     "variables": {
-        "uniqueSubDomainName": "[format('{0}-{1}', parameters('baseName'), parameters('endpointPrefix'))]"    
+        "uniqueSubDomainName": "[format('{0}-{1}', parameters('baseName'), parameters('endpointPrefix'))]",
+        "endpointValue": "[format('https://{0}-{1}{endpointSuffix}', parameters('baseName'), parameters('endpointPrefix'), parameters('endpointSuffix'))]" 
```

Comment:
should `{endpointSuffix}` be `{2}` ?  (I may not understand the syntax)

Reply:
Yes!